corehq/elastic.py

```diff
@@ -26,18 +26,20 @@ def get_es_new(**kwargs):
     Get a handle to the configured elastic search DB.
     Returns an elasticsearch.Elasticsearch instance.
     """
-    es_hosts = getattr(settings, 'ELASTICSEARCH_HOSTS', None)
-    if not es_hosts:
-        es_hosts = [settings.ELASTICSEARCH_HOST]
-
-    hosts = [
-        {
-            'host': host,
-            'port': settings.ELASTICSEARCH_PORT,
-        }
-        for host in es_hosts
-    ]
-    return Elasticsearch(hosts, **kwargs)
+    if not get_es_new._es_client:
```

Comment:
Should we also turn on connection sniffing periodically to make sure that the cluster/connection is still in a good state? http://elasticsearch-py.readthedocs.io/en/master/index.html#sniffing